draft-ietf-oauth-transaction-tokens.md

```diff
@@ -497,6 +500,14 @@ A requester MAY use a self-signed JWT as a `subject_token` value. In that case,
 
 The self-signed JWT MAY contain other claims.
 
+### Unsigned JSON Object Subject Token Type {#unsigned-json-subject-token-type}
+A requester MAY use an unsigned JSON object as a `subject_token` value. In that case, the requester MUST set the `subject_token_type` value to: `urn:ietf:params:oauth:token-type:unsigned_json`. The JSON object in the subject token MUST contain the following fields:
+
+* `sub`: The subject for whom the Txn-Token is being requested. The Txn-Token Service SHALL use this value in determining the `sub` value in the Txn-Token issued in the response to this request.
+* `exp`: The expiration time of the requested Txn-Token.
```

Comment:
Good point. I've added a line in the Txn-Token Request processing section to clarify.

Reply:
I think the exp should the the validity of the unsigned JSON object to keep it semantically equivalent to the self-signed JWT "The expiration time for the JWT."

Reply:
@gffletch Although this would be more in line with JWTs, I'm not sure if this is useful. The unsigned JSON is typically constructed by the workload right before performing the request and it's usuallly a one-off. The server would rread the unsigned JSON and issue the Txn-Token after which it just thows away the unsigned JSON. It may help to detect large clock skews between workload and TTS though.

Reply:
I just don't like changing the semantics of the `exp` field between different token types. What about describing the `exp` value as...

`exp` : The expiry time of the unsigned JSON object which the TTS MAY use as input to determining the life-time of the Txn-token.

If we need a way for the requesting workload to request an expiry time for the Txn-token then we should allow that for all flows and not just this one.

Reply:
Agreed, used your proposed language.